src/instructlab/cli/rag/ingest.py

```diff
@@ -0,0 +1,89 @@
+# Standard
+import logging
+
+# Third Party
+import click
+
+# First Party
+from instructlab import clickext
+from instructlab.defaults import DEFAULTS
+from instructlab.rag.taxonomy_utils import lookup_processed_documents_folder
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.option(
+    "--document-store-uri",
+    "uri",
+    type=click.STRING,
+    cls=clickext.ConfigOption,
+    config_class="rag",
+    config_sections="document_store",
+)
+@click.option(
+    "--document-store-collection-name",
+    "collection_name",
+    type=click.STRING,
+    cls=clickext.ConfigOption,
+    config_class="rag",
+    config_sections="document_store",
+)
+@click.option(
+    "--embedding-model-name",
+    "model_name",
+    type=click.STRING,
+    cls=clickext.ConfigOption,
+    config_class="rag",
+    config_sections="embedding_model",
+)
+@click.option(
+    "--input",
+    "input_dir",
+    required=False,
+    type=click.Path(
+        exists=True,
+        dir_okay=True,
+        file_okay=False,
+    ),
+    help="Directory where pre-processed documents are located.",
+)
+@click.pass_context
+@clickext.display_params
+def ingest(
+    ctx,
+    uri,
+    collection_name,
+    model_name,
+    input_dir,
+):
+    """The embedding ingestion pipeline"""
+    logger.debug(f"Document Store: {collection_name} @ {uri}")
+    logger.debug(f"Embedding model: {model_name}")
+
+    if input_dir is None:
+        output_dir = ctx.obj.config.generate.output_dir
+        if output_dir is None:
+            output_dir = DEFAULTS.DATASETS_DIR
```

Comment:
Minor suggestion... If the output directory is not defined in the config, it might be worthwhile to add a debug logger statement to let people know where things are being stored. Something like...

```suggestion
        if output_dir is None:
            output_dir = DEFAULTS.DATASETS_DIR
            logger.debug(f"`output_dir` not defined in config. Using default path {output_dir}")
```